ios/CodePush/CodePush.m

```diff
@@ -47,6 +48,11 @@ @implementation CodePush {
 
 #pragma mark - Public Obj-C API
 
++ (NSURL *)binaryBundleURL
```

Comment:
It is used in both CodePushPackage and CodePushUpdateUtils to get the binaryURL to 1. calculate the hash 2. apply diff updates